vuetify-demo/src/components/Navbar.vue

```diff
@@ -0,0 +1,53 @@
+ <template>
+  <div>
+    <v-app-bar color="deep-black accent-4" dense dark min-width="300">
+      <v-toolbar-title>{{ showRoomName }}</v-toolbar-title>
+
+      <v-spacer></v-spacer>
+
+      <v-btn plain v-if="this.isAuth" @click="openModal">Add Car</v-btn>
+
+      <v-btn plain v-if="this.isAuth">Home</v-btn>
+      <router-link to="/login">
+        <v-btn plain v-if="!this.isAuth"> Login</v-btn>
+      </router-link>
+      <router-link to="/register">
+        <v-btn plain v-if="!this.isAuth"> Register</v-btn>
+      </router-link>
```

Comment:
There is no need to use <router-link> while using Vuetify. You can directly use "to" props in <v-btn>.